go/README.md

```diff
@@ -1,15 +1,18 @@
 # Go
 
-This is a basic example app that uses Go to load the Intercom Messenger.
+This repo is an example application of the Intercom Messenger on a Go web app. You can follow along and look at how it is implemented in the code.
 
-For the purposes of demonstrating how the Intercom Messenger can be loaded differently for visitors and logged in users this app offers the option to create an account and login.
+On the **server side**, return identifying information about the currently logged-in user to the Intercom Messenger:
+1. Edit the placeholders to include the email address of the user who is currently logged-in ([see code](https://github.com/intercom/intercom-install-examples/blob/14465ba85d0cfbca63e12a7e72efa1c2294d0d58/go/app.go#L37)). You will need to return this data from your server ([see code](https://github.com/intercom/intercom-install-examples/blob/14465ba85d0cfbca63e12a7e72efa1c2294d0d58/go/app.go#L41)).
+1. Now, generate a hash based on the user’s email and your Intercom identity verification secret key ([see code](https://github.com/intercom/intercom-install-examples/blob/14465ba85d0cfbca63e12a7e72efa1c2294d0d58/go/app.go#L39)). If you want more info on why we do this check out [this doc](https://docs.intercom.com/configure-intercom-for-your-product-or-site/staying-secure/enable-identity-verification-on-your-web-product).
+
+On the **client side**:
+1. Paste the code right before the closing body tag of every page where you want the Intercom Messenger to appear ([see code](https://github.com/intercom/intercom-install-examples/blob/14465ba85d0cfbca63e12a7e72efa1c2294d0d58/go/layout.html#L16)).
+
+*If you have already installed the Intercom Messenger, you only have to update the `window.intercomSettings` with the `user_` attributes for logged-in users.*
```

Comment:
I think you need to add 4 spaces at the beginning of this line and remove the spaces above/below for it to render as a sub bullet in this list.

Reply:
This guide here says have a blank space above https://github.com/intercom/intercom-install-examples/pull/9